cmd/cloudshell_open/locations.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+func listLocations() ([]string, error) {
+	loc := []string{
```

Comment:
We shouldnt hardcode. There’s a locations api on cloud run. But that will require bringing in GCP go libraries.

Reply:
Can we get them from `gcloud` or via REST?

Reply:
I don't think so + it would be slower.
Similarly we can also get the projects via API once we import the package.